Sim/SimPapas/src/PapasSimulatorTool.h

```diff
@@ -17,7 +17,7 @@ class MCParticleCollection;
 /** @class PapasSimulatorTool Sim/SimPapas/src/PapasSimulatorTool.h PapasSimulatorTool.h
  *
  *  Runs the papas simulation algorithm. This creates a list of particles from Pythia generated particles.
- *  It then creates particle flow simulated and true and smeared clusters and tracks. All of these
+ *  It then creates particle-flow-simulated true and smeared clusters and tracks. All of these
```

Comment:
particle flow does the reconstruction from the tracks and the clusters. 
it does not simulate them, and does not play a role in the creation of true and smeared clusters. 

replace by: 

It then simulates the interaction of these particles in the detector to produce true and smeared clusters and tracks, that will be used in the particle flow reconstruction.

Reply:
done